src/components/export/ExportSection.jsx

```diff
@@ -48,7 +48,7 @@ class ExportSection extends Component {
     const { displayModal } = this.state
     return (
       <div>
-        <div className={viewStyles['set-view-section']}>
+        <div className="{viewStyles['set-view-section']} u-mb-1">
```

Comment:
```jsx 
import classNames from 'classnames'
```

```suggestion
        <div className={classNames(viewStyles['set-view-section'], 'u-mb-1')}>
```

Reply:
Why this change? It's more verbose but I don't understand what we get in exchange

Reply:
I read the diff from the github and didn't see the closing bracket, so I thought that it should not work. 

Beside the fact it's easier for me to read, we get nothing.